### ISSUE TITLE
Help needed with Authentication

### ISSUE THREAD

**Issue by sirib** [andrewmcgivery/angular-soap#14]
Hi,
I am trying to access list of busstops from this address
http://st.atb.no/infotransit/userservices.asmx?WSDL
this needs authentication to access.

I follwed the "Example 5: Set Connection Credentials" procedure and still no success.

It throws floowing error:

XHR finished loading: GET "http://st.atb.no/infotransit/userservices.asmx?wsdl".

POST http://st.atb.no/infotransit/userservices.asmx 500 (Internal Server Error)

SOAPClient._sendSoapRequest @ soapclient.js:190
SOAPClient._onLoadWsdl @ soapclient.js:153
xmlHttp.onreadystatechange @ soapclient.js:142

XHR finished loading: POST "http://st.atb.no/infotransit/userservices.asmx".

**Comment by andrewmcgivery** [comment]
Have you double checked Chrome Dev Tools to make sure it is sending the request as you expect it to?

**Comment by sirib** [comment]
yes i double checked..the error i quoted here is from Chrome Devtools

**Comment by sirib** [comment]
Hi, i still could not figure out a way fix this issue. Any help would be appreciated.
Thanks in advance

**Comment by viveknandhan527** [comment]
Hi andrew,
I am also same problem. Thanks in advance
